### ISSUE TITLE
nixos.tests.ipv6.x86_64-linux fails blocking nixos channels

### ISSUE THREAD

**Issue by FRidh** [NixOS/nixpkgs#76710]
**Describe the bug**
https://hydra.nixos.org/build/108819057

```
Global addresses can be obtained, pinged, and reached via http
client: waiting for success: [ `ip -o -6 addr show dev eth1 scope global -tentative -temporary | wc -l` -eq 1 ]
client # [   14.204119] dhcpcd[732]: eth1: Router Advertisement from fe80::5054:ff:fe12:102
router # [   25.581308] radvd[649]: [Dec 30 10:00:44] radvd (649): IPv6 forwarding on interface seems to be disabled, but continuing anyway
router # [   27.281146] nscd[705]: 705 checking for monitored file `/etc/netgroup': No such file or directory
client # [   27.793566] nscd[716]: 716 checking for monitored file `/etc/netgroup': No such file or directory
server # [   28.025739] nscd[715]: 715 checking for monitored file `/etc/netgroup': No such file or directory
error: action timed out
(912.84 seconds)
```

**Comment by FRidh** [comment]
cc @mmilata

**Comment by andersk** [comment]
`git bisect` says the first bad commit is 0bc45ecba1391878dc6f3f2560caceb669d147e6 “dhcpcd: 8.1.2 -> 8.1.4” (#76224).

**Comment by andersk** [comment]
And `git bisect` on dhcpcd says the first bad commit is rsmarples/dhcpcd@569051c8aa8fc297eb8edb7bd228e0fd353d30c1 “Linux: prefer ms RA times”.

**Comment by FRidh** [comment]
cc @nlewo

**Comment by andersk** [comment]
Sent a proposed fix upstream as rsmarples/dhcpcd#12.

**Comment by andersk** [comment]
And sent the fix here as #76798.
